packages/node/src/protocol/uninstall.ts

```diff
@@ -121,7 +126,13 @@ async function proposeStateTransition(
     provider
   );
 
-  const newStateChannel = sc.uninstallApp(appIdentityHash, increments);
+  const newStateChannel = sc.uninstallApp(
+    appIdentityHash,
+    increments,
+    // installing/uninstalling apps with ERC20 is not yet supported
+    // so all installs/uninstalls default to ETH
```

Comment:
This comment makes no sense

Why does the function take a token parameter

Reply:
When an app gets installed, its params will need to indicate what asset it wants to use for collateral. This asset will be used to update the free balance when both doing an install and uninstall.

Since currently installing/uninstalling apps for ERC20 tokens is not supported, this defaults to ETH. When the PR for installing/uninstalling apps for ERC20 tokens is up, this will be modified to take the correct asset into account.

Reply:
The function should not take a parameter it does not intend on using